packages/flutter/test/widgets/routes_test.dart

```diff
@@ -1782,7 +1845,9 @@ class _TestDialogRouteWithCustomBarrierCurve<T> extends PopupRoute<T> {
     required Widget child,
     this.barrierLabel,
     Curve? barrierCurve,
+    Color? barrierColor,
```

Comment:
why not just `this.barrierColor = Colors.black`
and then define
`Color barrierColor;`
with this you don't need to override the getter.